lib/portlayer/portlayer.go

```diff
@@ -83,5 +84,46 @@ func Init(ctx context.Context, sess *session.Session) error {
 		return err
 	}
 
+	unbindSerialPorts(sess)
+
 	return nil
 }
+
+// unbindSerialPorts disconnects serial ports backed by network on the VCH's old IP
+// for running containers. This is useful when the appliance or the portlayer restarts
+// and the VCH has a new IP. Any errors are logged and portlayer init proceeds as usual.
+func unbindSerialPorts(sess *session.Session) {
+	// Get all running containers from the cache
+	runningState := new(exec.State)
+	*runningState = exec.StateRunning
+	containers := exec.Containers.Containers(runningState)
```

Comment:
containers := exec.Containers.Containers(&exec.StateRunning) should work here instead those, no?